src/remote/impl/identity.service.ts

```diff
@@ -51,4 +51,18 @@ export class IdentityService implements IIdentityService {
         };
         return this.http.requestWithRetry(request);
     }
+
+    /**
+     * Queries the identity service to get the finger positions with the best quality scores
+     * TODO the identity service should update the positions endpoint to accept data in the body instead of via url params
+     */
+    public async qualityCheck(filters: any): Promise<any> {
+        const filterKey = Object.keys(filters)[0];
+        const params = `${filterKey}=${filters[filterKey]}`;
```

Comment:
do these values need to be escaped since they are used in a url?

Reply:
Good call - short run I think we should be ok because the filters shouldn't need url escaping, and long term we want to switch the identity service to use body json rather than url params since it's simpler. I added a ticket to switch the identity service side, but if we decide not to do that, I'll url escape the callers of the positions endpoint.